examples/prometheus.yml

```diff
@@ -0,0 +1,39 @@
+global:
+  scrape_interval: 15s
+  scrape_timeout: 10s
+  evaluation_interval: 15s
+alerting:
+  alertmanagers:
+  - static_configs:
+    - targets: []
+    scheme: http
+    timeout: 10s
+    api_version: v1
+rule_files:
+- /etc/prometheus/alerts.yml
+scrape_configs:
+- job_name: prometheus
+  honor_timestamps: true
+  scrape_interval: 15s
+  scrape_timeout: 10s
+  metrics_path: /metrics
+  scheme: http
+  static_configs:
+  - targets:
+    - localhost:9090
+    - localhost:7979
+- job_name: json_exporter
+  honor_timestamps: true
+  scrape_interval: 5s
+  metrics_path: /probe
+  scheme: http
+  static_configs:
+  - targets:
+    - localhost:7979
+  params:
+    target: ["http://localhost:8000/examples/data.json"]
+  relabel_configs:
+    - source_labels: [__param_target]
+      target_label: endpoint
+      action: replace
```

Comment:
An exemples à la blackbox_exporter might be better? With the json as target

Reply:
Maybe I didn't understand this fully I think.
Current example set up: 
`prometheus` -> scrapes `json_exporter` -> fetches `localhost:8000/examples/data.json`
Proposed example set up:
`prometheus` -> scrapes `blackbox_exporter` -> probes `json_exporter`? -> fetches `localhost:8000/examples/data.json`?
:thinking:

Reply:
No, it's more about how you use Prometheus configs.

```
...
    static_configs:
      - targets:
        - http://host-1:8000/examples/data.json
        - http://host-2:8000/other-examples/data.json
    relabel_configs:
      - source_labels: [__address__]
        target_label: __param_target
      - source_labels: [__param_target]
        target_label: instance
      - target_label: __address__
        replacement: 127.0.0.1:7979  # The json exporter's real hostname:port.
```

Reply:
Thanks for the clarification, I have updated the example prometheus config. Let me know if the updated one looks good.